src/export/ExportWavPack.cpp

```diff
@@ -388,7 +389,7 @@ ProgressResult ExportWavPack::Export(AudacityProject *project,
             const float *mixed = reinterpret_cast<const float*>(mixer->GetBuffer());
             for (decltype(samplesThisRun) j = 0; j < samplesThisRun; j++) {
                for (size_t i = 0; i < numChannels; i++) {
-                  int64_t intValue = (*mixed++) * (1u << 31);
+                  int64_t intValue = (*mixed++) * (std::numeric_limits<int32_t>::max());
```

Comment:
Minor, but I expect a compiler warning here. Let's add `static_cast<int64_t>` here. Sorry I've missed that before...